src/packaging/kpatch/MANIFEST.yaml

```diff
@@ -60,6 +64,9 @@ tarballs:
             - VDO_UPSTREAM
           +postProcessor: ../github/removeInternal.sh
         src/c++/uds/kernelLinux/uds:
+          +excludes:
+            - uds-sysfs.c
+            - uds-sysfs.h
           +defines:
             - VDO_UPSTREAM
```

Comment:
Not wrong, but this will conflict with the indexer subdirectory work. Just to keep it in mind.